rust/core-lib/src/file.rs

```diff
@@ -136,12 +178,23 @@ impl FileManager {
     fn save_new(&mut self, path: &Path, text: &Rope, id: BufferId) -> Result<(), FileError> {
         try_save(path, text, CharacterEncoding::Utf8)
             .map_err(|e| FileError::Io(e, path.to_owned()))?;
+
+        #[cfg(feature = "notify")]
+        let new_tail_details = TailDetails {
```

Comment:
This looks like you could `derive(Default)` for this type, and then just do `TailDetails::default()`.

Reply:
Done.